perf/security/sds-tests/citadel-1/cleanup.sh

```diff
@@ -4,8 +4,9 @@
 
 set -xe
 NAMESPACE=${NAMESPACE:?"specify the namespace to delete"}
+CLUSTER=${CLUSTER:?"specify the cluster for running the test"}
```

Comment:
Can this just default to the current context?

Reply:
This is not default to the current context as there may be multiple tests running on multiple clusters and cleaning up the current context may accidentally clean up a wrong cluster. Hence the cluster is explicately asked for.